tests/kernel/shutdown_ltp.pm

```diff
@@ -41,7 +41,11 @@ sub run {
         upload_logs($ver_linux_log, failok => 1);
     }
 
-    script_run('[ "$ENABLE_WICKED" ] && systemctl enable wicked');
+    # dhclient requires no wicked service not only running but also disabled
+    # TODO: chack can be safely removed next release after 20190930 is packaged for QAM
+    if (script_run('grep -q "systemctl restart wicked" /opt/ltp/testcases/bin/dhcp_lib.sh') eq 1) {
```

Comment:
since it seems we are using this condition more than 1 time, would it make sense to move it to some ltp utils or something?